appcues/publish-maven.gradle

```diff
@@ -19,7 +19,7 @@ private String generateVersionName() {
     String classifier = appcuesProperties.getProperty("VERSION_CLASSIFIER")
     String versionName = "${major}.${minor}.${patch}"
 
-    if (classifier != null) {
+    if (classifier?.trim()) {
```

Comment:
this checks null or empty - we had empty before leading to a `1.0.0-` version name being used in the maven publish